Recrutify/Recrutify.Services/EmailModels/EmailRequest.cs

```diff
@@ -1,11 +1,15 @@
-﻿namespace Recrutify.Services.EmailModels
+﻿using System.Text;
+
+namespace Recrutify.Services.EmailModels
 {
     public class EmailRequest
     {
         public string ToEmail { get; set; }
 
         public string Subject { get; set; }
 
-        public string Body { get; set; }
+        public string HtmlBody { get; set; }
+
+        public StringBuilder FileBody { get; set; }
```

Comment:
May be AttachmentBody, and not string builder - string

Reply:
+